source/level/TestGenState.hx

```diff
@@ -43,28 +43,27 @@ class TestGenState extends FlxState {
 
 		FlxG.collide(playerTank, map);
 		FlxG.collide(enemyTanks, map);
-		FlxG.collide(dynamicTank, map);
-		FlxG.collide(pursuitTank, map);
 
 		checkFinished();
 	}
 
 	private function addTanks() {
 		var tankCoordinates = [250, 300, 350];
 		playerTank = TankFactory.NewPlayerTank(500, 500);
-		dynamicTank = TankFactory.NewDynamicTank(50, 100, playerTank, map);
-		pursuitTank = TankFactory.NewPursuitTank(200, 400, playerTank, map);
 
-		enemyTanks = new FlxTypedGroup<Tank>(3);
+		enemyTanks = new FlxTypedGroup<Tank>();
+		var dynamicTank = TankFactory.NewDynamicTank(50, 100, playerTank, map);
+		var pursuitTank = TankFactory.NewPursuitTank(200, 400, playerTank, map);
+
 		for (x in tankCoordinates) {
 			var enemy = TankFactory.NewDumbTank(x, 50);
 			enemyTanks.add(enemy);
 		}
 
+		enemyTanks.add(dynamicTank);
+		enemyTanks.add(pursuitTank);
```

Comment:
I think we're missing the `.getAllSprites()` call for these `adds` - probably means these don't have cannons visible.

Reply:
I believe the loop on line 67 takes care of those .getAllSprites since dynamic and pursuit are contained in the EnemyTanks group